### PR TITLE
Chores: Multiple tweaks

### PR DESCRIPTION
**Pull request by Thomzoy**
This PR fixes and update a lot of project-related stuffs

- Adding jobs in the `Testing` CI stage

   - Runs `pre-commit` for linting
   - Builds documentation

- Adding Issues and PR template
- Updating the LICENCE file
- Updating the `pyproject.toml` metadata
- Removing last links to previous GitLab repo
- Changing the REAME image `src` to an absolute path so it would work on PyPI
- Adding back a `Biology summary` folder in the doc that wa excluded because of a `.gitignore` rule
- Updating changelog format

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -55,7 +55,9 @@ packages = ["eds_scikit"]
 version = {attr = "eds_scikit.__version__"}
 
 [project.urls]
-source = "https://github.com/aphp/EDS-Scikit"
```
Comment (Vincent-Maladiere) [inline pyproject.toml:58]: Homepage should be the same as documentation in our case, and Source / Repository should be defined. Cf https://peps.python.org/pep-0621/#example

File: docs/functionalities/biology/Biology_summary/Custom_entity/stats_summary.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+LOINC_concept_code,AnaBio_concept_code,LOINC_concept_name,AnaBio_concept_name,unit_source_value,count,mean,std,min,25%,50%,75%,max,MAD,max_threshold,min_threshold
```
Comment (Vincent-Maladiere) [inline docs/functionalities/biology/Biology_summary/Custom_entity/stats_summary.csv:1]: Are these csv and html supposed to be there?

**Review by Vincent-Maladiere — CHANGES_REQUESTED** [review]
Great work @Thomzoy! Some request changes and we're good to go

File: pyproject.toml (hunk 1 above)
Comment (Thomzoy) [reply pyproject.toml:58]: I used these keywords to display those links nicely in PyPI (see [here](https://github.com/pypi/warehouse/blob/01f87cfdb6275a8c8bc11b838618df246fc10fec/warehouse/templates/packaging/detail.html#L22))

File: docs/functionalities/biology/Biology_summary/Custom_entity/stats_summary.csv (hunk 2 above)
Comment (Thomzoy) [reply docs/functionalities/biology/Biology_summary/Custom_entity/stats_summary.csv:1]: Yep, they come from fake data and were erroneously removed !

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+## Description
+
+<!--- Describe the changes. -->
+
+## Checklist
+
+<!--- Every item must be checked before the PR is merged. [] -> [x] -->
+
+- [ ] If this PR is a bug fix, the bug is documented in the test suite.
+- [ ] Changes were documented in the changelog (pending section).
```
Comment (Thomzoy) [inline .github/pull_request_template.md:12]:
```suggestion
- [ ] Changes were documented in the changelog (pending section).
- [ ] If relevant, appropriate tests were implemented in the test suite
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+- [ ] If necessary, changes were made to the documentation (eg new pipeline).
```
[the remark this answers is not in the document]
Comment (Thomzoy) [reply .github/pull_request_template.md:13]: Added ;)

File: pyproject.toml (hunk 1 above)
Comment (Vincent-Maladiere) [reply pyproject.toml:58]: I still think "Source Code" is needed

https://github.com/scikit-learn/scikit-learn/blob/07958e66a146e015bd738c9c7e3be41ef5ab6097/setup.py#L45

https://pypi.org/project/scikit-learn/

Comment (Thomzoy) [reply pyproject.toml:58]: We have the "Download File" button just above, I think I'll do the job !

**Comment by Vincent-Maladiere** [comment]
Alright, let's merge when the CI is green :)

**Comment by Thomzoy** [comment]
:rocket:

**Pull request merged**